src/SelfServe/SelfServeTelemetryProcessor.ts

```diff
@@ -1,69 +1,24 @@
-import { sendMessage } from "../Common/MessageHandler";
```

Comment:
Alternative suggestion. You could just inspect the URL to see if it contains `selfServe` inside the existing telemetry processor and then send the correct type that way. Then you would not need this file and you can use the same exact API we have now.

Reply:
discussed offline, going with this approach for now since we need to enforce the SelfServeTelemetryData to have a mandatory "SelfServeClassName" field.